### ISSUE TITLE
Group instanced models support

### ISSUE THREAD

**Issue by samvila** [godotengine/godot-blender-exporter#130]
At this moment the exporter doesn't consider any object instanced, to reproduce this just create a cube and add it into a group, then add some group instances of the cube on the scene and exporte the scene into godot with the exporter. You will notice that in Godot shows the group instances as Spatial 3D nodes but they are all empty.

This feature will be highly beneficial to have it specially for scenes with repeated objects and to save memory by having instanced models instead of real geometry during runtime.

**Comment by Jason0214** [comment]
yep, there is no support for group instance for now and it would be beneficial to have it supported. Thank you for the suggestion!

Actually the exporter is well making use of instance, for example if you 'copy' a mesh with **Alt+d**, the two meshes would share the same mesh resource. But group instance is kind of different thing.
